### PR TITLE
[14.0] Misc fixes

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

**Review by jcoux — APPROVED** [review]

File: connector_search_engine/models/se_binding.py [hunk 1]
```diff
@@ -190,11 +190,11 @@ def synchronize(self):
         # and to do the right action as we are in a transaction.
         export_ids = []
         delete_ids = []
-        for work in self._work_by_index():
+        for work in self.sudo()._work_by_index():
```
Comment (sebastienbeau) [inline connector_search_engine/models/se_binding.py:193]: What is your case when you need a sudo ?

Comment (sebastienbeau) [reply connector_search_engine/models/se_binding.py:193]: @simahawk

Comment (simahawk) [reply connector_search_engine/models/se_binding.py:193]: @sebastienbeau for the same reason described here https://github.com/OCA/search-engine/blob/14.0/connector_search_engine/models/se_binding.py#L134-L139

Comment (sebastienbeau) [reply connector_search_engine/models/se_binding.py:193]: Ok

**Review by sebastienbeau — APPROVED** [review]
Lgtm (code review)

**Pull request merged**